src-vue/src/i18n/lang/de.json

```diff
@@ -0,0 +1,156 @@
+{
+    "menu": {
+        "play": "Spielen",
+        "changelog": "Versionhistorie",
+        "mods": "Mods",
+        "settings": "Einstellungen",
+        "dev": "Dev"
+    },
+
+    "generic": {
+        "yes": "Ja",
+        "no": "Nein",
+        "error": "Fehler",
+        "cancel": "Abbrechen",
+        "informationShort": "Info"
+    },
+
+    "play": {
+        "button": {
+            "northstar_is_running": "Spiel läuft",
+            "select_game_dir": "Titanfall2 ordner wählen",
+            "install": "Installieren",
+            "installing": "Installiert...",
+            "update": "Aktualisieren",
+            "updating": "Aktualisiert...",
+            "ready_to_play": "Spiel starten"
+        },
+
+        "unknown_version": "Unbekannte Version",
+        "see_patch_notes": "Siehe Patch-Notizen",
+        "players": "Spieler",
+        "servers": "Server",
+        "unable_to_load_playercount": "Spielerzahl konnte nicht geladen werden",
+        "northstar_running": "Northstar läuft:",
+        "origin_running": "Origin läuft:"
+    },
+
+    "mods": {
+        "local": {
+            "no_mods": "Keine Mods gefunden.",
+            "delete_confirm": "Bist du dir sicher, dass du diesen Mod löschen möchtest?",
+            "delete": "Löschen",
+            "part_of_ts_mod": "Dieser Northstar Mod ist teil eines Thunderstore Mods",
+            "success_deleting": "{modName} erfolgreich gelöscht"
+        },
+
+        "online": {
+            "no_match": "Keine passenden Mods gefunden.",
+            "try_another_search": "Versuche eine andere Suchanfrage!"
+        },
+
+        "menu": {
+            "local": "Lokal",
+            "online": "Online",
+            "filter": "Filter",
```

Comment:
Forgotten translation, or do you use the same word in German? :)

Reply:
Actually the same word in German, ye ^^